ui/lib/sync/addon/components/secrets/destination-header.ts

```diff
@@ -26,11 +26,11 @@ export default class DestinationsTabsToolbar extends Component<Args> {
   async deleteDestination() {
     try {
       const { destination } = this.args;
-      const message = `Successfully deleted destination ${destination.name}.`;
+      const message = `Destination ${destination.name} has been queued for deletion.`;
       await destination.destroyRecord();
       this.store.clearDataset('sync/destination');
       this.router.transitionTo('vault.cluster.sync.secrets.destinations');
```

Comment:
Does it make sense to remove the transition so the user has a chance to view the purge information? It's a bit weird codewise since we would have to reload the destination right after a destroy.

Reply:
I think we can update this transition to the destination details, like you suggested below! Yankun agreed with that

Reply:
Circling back here - is this suggestion still relevant?

Reply:
Not since it has been updated to transition to destination secrets. All good.